app/views/test_executions/_test_info.html.erb

```diff
@@ -6,14 +6,20 @@
 
 <div class = 'col-sm-4'>
   <h1><%= info_title_for_product_test(task.product_test) %></h1>
-  <% unless task.product_test.is_a? ChecklistTest %>
+  <% if (task.product_test.is_a? MeasureTest) || (task.product_test.is_a? FilteringTest) %>
     <strong>Measure: </strong><span><%= task.product_test_name %></span><br/>
-  <% end %>
-  <strong>HQMF ID: </strong><span><%= task.measure_ids.join(',') %></span><br/>
```

Comment:
It appears this line has been removed for the case where the test is a checklist test. Did we intend on removing the HQMF id from the "Record Sample Test Information" page?

Reply:
That was unintentional.  Thou...I'm not sure how useful having the HQMF id on the Record Sample Test Execution. (below is a screenshot from the demo server with the HQMF ids)

<img width="950" alt="image" src="https://user-images.githubusercontent.com/8173551/55909413-f3d60180-5ba9-11e9-87b3-2b154fbd8f2b.png">